test/integration/multikueue/setup_test.go

```diff
@@ -407,3 +415,532 @@ var _ = ginkgo.Describe("MultiKueue", ginkgo.Ordered, ginkgo.ContinueOnFailure,
 		})
 	})
 })
+
+var _ = ginkgo.Describe("MultiKueueDispatcherIncremental", ginkgo.Ordered, ginkgo.ContinueOnFailure, func() {
```

Comment:
Please move the new tests to `dispatcher_test.go`. It will make it easier to find the tests

Reply:
Done